scripts/packaging/package-marketplace.snapshot

```diff
@@ -8,6 +8,3367 @@ Resources/
 Resources/Icon128.png
 Scripts/
 sentry-cli.properties
+sentry-native/
+sentry-native/.clang-format
+sentry-native/.codechecker-ignore
+sentry-native/.codecov.yml
+sentry-native/.craft.yml
+sentry-native/.editorconfig
+sentry-native/.git
```

Comment:
Maybe a @tustanivsky question:

Should we be adding `.git` to the packages? This will include all commit history etc.
Similarly about `.github`, looks like we're adding the whole git submodule as-is.